src/main/java/org/apache/commons/imaging/formats/png/PngWriter.java

```diff
@@ -447,7 +447,7 @@ public void writeImage(final BufferedImage src, final OutputStream os, Map<Strin
             final PaletteFactory paletteFactory = new PaletteFactory();
 
             if (hasAlpha) {
-                palette = paletteFactory.makeQuantizedRgbaPalette(src, hasAlpha, maxColors);
+                palette = paletteFactory.makeQuantizedRgbaPalette(src, true, maxColors);
```

Comment:
I think it's fine to pass `hasAlpha` here. Easier to see (or guess) that that's a flag for transparency, especially in case we add more booleans to that method later.

Reply:
HI @kinow 
Changed
TY